### DISCUSSION TITLE
Confirm behavior of --quiet in script(1)

### DISCUSSION THREAD

**Discussion by ahmedshash** [util-linux/util-linux#1736] [Q&A] [answered]
Can you please confirm the behavior of --quiet in script(1). I can see that it doesn't output "Script started/done" [if set](https://github.com/util-linux/util-linux/blob/master/term-utils/script.c#L939-L940), but should it also include the "Script started" header on [line 399](https://github.com/util-linux/util-linux/blob/master/term-utils/script.c#L399) and "Script done" footer on [lines 298-302](https://github.com/util-linux/util-linux/blob/master/term-utils/script.c#L298-L302)?

Many Thanks

**Comment by karelzak** [comment, marked as answer]
Yes, the log file is not affected by the  --quiet option. 

If I remember, in old versions, the option had an impact on the log files too, but now the headers contain some extra information, so it keeps it in the log file independently on --quiet.

**Reply by ahmedshash** [reply]
Thank you for confirming
